qiskit/synthesis/one_qubit/one_qubit_decompose.py

```diff
@@ -76,8 +76,8 @@ class OneQubitEulerDecomposer:
     parameters :math:`(\theta, \phi, \lambda)`, and a phase parameter
     :math:`\gamma`. The value of the parameters for an input unitary depends
     on the decomposition basis. Allowed bases and the resulting circuits are
-    shown in the following table. Note that for the non-Euler bases (U3, U1X,
-    RR), the ZYZ Euler parameters are used.
+    shown in the following table. Note that for the non-Euler bases :math:`(U3, U1X,
+    RR)`, the :math:`ZYZ` Euler parameters are used.
```

Comment:
The first of these may not need to be written in mathmode - it's not super clear to me.